RevenueCatUI/Templates/V2/ViewHelpers/BackgroundStyle.swift

```diff
@@ -20,7 +20,6 @@ enum BackgroundStyle {
 
     case color(PaywallComponent.ColorScheme)
     case image(PaywallComponent.ThemeImageUrls)
-    case gradient
```

Comment:
Following the spec, gradient is not a different type of background but is encoded as part of the `ColorInfo` struct.